partner_industry_secondary/views/res_partner_view.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015 Antiun Ingenieria S.L. - Javier Iniesta
+     Copyright 2018 Eficent Business and IT Consulting Services, S.L. - Jordi Ballester
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+    <record model="ir.ui.view" id="view_partner_form_industry">
+        <field name="name">Partner form with industry</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <field name="industry_id" position="after">
+                <field name="show_partner_industry_for_person" invisible="1" />
```

Comment:
```suggestion
```
You can remove invisible fields based on this PR: https://github.com/odoo/odoo/pull/137031

Reply:
and the same may apply to the changes below.